torch/_inductor/lowering.py

```diff
@@ -2185,6 +2185,8 @@ def is_aligned_backward(x):
 make_fallback(aten.pixel_unshuffle)
 make_fallback(aten.polygamma)
 make_fallback(aten.put)
+make_fallback(aten.reflection_pad1d)
+make_fallback(aten.replication_pad1d)
```

Comment:
I think your rebase has gone awry.

Reply:
You need to remove the `aten.put` fallback since we're adding a decomposition that replaces it.